Kernel/TimerQueue.cpp

```diff
@@ -16,12 +16,12 @@ namespace Kernel {
 static Singleton<TimerQueue> s_the;
 static Spinlock<LockRank::None> g_timerqueue_lock {};
 
-Time Timer::remaining() const
+Duration Timer::remaining() const
 {
     return m_remaining;
 }
 
-Time Timer::now(bool is_firing) const
+Duration Timer::now(bool is_firing) const
```

Comment:
suspicious return from `current_time` as `Duration`… i guess this should return now `UnixDateTime`, right?